### PR TITLE
adding mainnet url in env

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -13,7 +13,7 @@ PORT=2015
 LOG_LEVEL=debug
 
 NETWORK_WS_URL=wss://archive.testnet.cere.network:9945
-NETWORKS=[{"URL": "wss://archive.testnet.cere.network:9945", "FAUCET": {}, "PASSWORD":"", "NETWORK": "TESTNET"}, {"URL": "wss://archive.testnet.dev.cere.network:9945", "FAUCET": {}, "PASSWORD":"", "NETWORK": "TESTNET_DEV"}]
+NETWORKS=[{"URL": "wss://archive.mainnet.stage.cere.network:9945", "FAUCET": {}, "PASSWORD":"", "NETWORK": "MAINNET"},{"URL": "wss://archive.testnet.cere.network:9945", "FAUCET": {}, "PASSWORD":"", "NETWORK": "TESTNET"}, {"URL": "wss://archive.testnet.dev.cere.network:9945", "FAUCET": {}, "PASSWORD":"", "NETWORK": "TESTNET_DEV"}]
```
Comment (AndreiNavoichyk) [inline .env.local:16]: Use `MAINNET_STAGE` here and everywhere

File: .env.local [hunk 2]
```diff
@@ -13,7 +13,7 @@ PORT=2015
...
 MAX_BALANCE=1
 NUMBER_OF_TOKENS_TO_SEND=1000000000000000000
```
Comment (AndreiNavoichyk) [inline .env.local:18]: Number of tokens should be updated. It should be 1000 (decimals is 10)

**Review by AndreiNavoichyk — APPROVED** [review]

File: .env.local (hunk 1 above)
Comment (karthikeyan-cere) [reply .env.local]: updated.

File: .env.local (hunk 2 above)
Comment (karthikeyan-cere) [reply .env.local]: Updated.

**Pull request merged**